src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -492,7 +551,7 @@
       <LogicalName>System.Data.SqlClient.SqlMetaData.xml</LogicalName>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup  Condition="'$(TargetGroup)' == 'netcoreapp'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
```

Comment:
Nit: Indentation

Reply:
Removed extra space in alignment with other tags.